### ISSUE TITLE
Copilot outdated, doesn't know about yolov8

### ISSUE THREAD

**Issue by LizardWizard777** [microsoft/vscode-copilot-release#910]
It perfectly assisted me with install/issues on yolov5, but on [yolov8](https://github.com/ultralytics/ultralytics) it couldn't help:

> I apologize for any confusion, but as of my last training data, YOLOv8 does not exist. My training data includes information up to YOLOv5. If Ultralytics has indeed released YOLOv8, my current knowledge does not include information about this version.

>As an AI, my knowledge is based on the training data I was last provided. I don't have the ability to access or retrieve real-time information from the internet or external databases. Therefore, I recommend referring to the official documentation or trusted sources for the most accurate and up-to-date information about YOLOv8.

**Comment by fisforfaheem** [comment]
Same for Flutter, I am getting answers from 2021 and 2022 :(

**Comment by deathvulture** [comment]
Same for node and typescript

**Comment by zivshek** [comment]
Yup, as of right now, Flutter version is 3.19.5, and the chat confidently responded "It seems there's a misunderstanding. Flutter's latest stable version at the time of writing is 2.8.1. If you're referring to Dart SDK version as 3.19.5, that's also not correct as Dart's latest stable version is 2.15.1." 

When are we going to feed it new data? For frontend technologies, outdated info is totally useless.

**Comment by fisforfaheem** [comment]
Dear team, kindly update the flutter and dart knowledge so we can benefit.


Stop posting new stuff in blogs instead try to update with these fixes

**Comment by rzhao271** [comment]
Hi, we have made a lot of changes over the past several months, and the newer models have improved as well. If you are still encountering this problem on the latest VS Code and Copilot Chat extension, please file a new issue in the [microsoft/vscode](https://github.com/microsoft/vscode) repository.
